examples/kokkos/gg_ttgg_auto/src/HelAmps_sm.cu

```diff
@@ -0,0 +1,808 @@
+//==========================================================================
+// This file has been automatically generated for C++ Standalone by
+// MadGraph5_aMC@NLO v. 2.7.3.py3, 2020-06-28
+// By the MadGraph5_aMC@NLO Development Team
+// Visit launchpad.net/madgraph5 and amcatnlo.web.cern.ch
+//==========================================================================
+
+#include "HelAmps_sm.h"
+#include <complex> 
+#include <cmath> 
+#include <iostream> 
+#include <cstdlib> 
+#include <thrust/complex.h> 
+
+namespace MG5_sm 
+{
+
+__device__ void ixxxxx(constdouble * pvec, double fmass, int nhel, int nsf, 
```

Comment:
I was being nosy and looking through your conversion as I was working through mine, noticed `constdouble` here is probably a typo